### PR TITLE
feat: deckgl for vue example

### PR DESCRIPTION
**Pull request by AdriSolid**
For: https://app.clubhouse.io/cartoteam/story/133650/vue-js-integration-example

### CODE REVIEW

**Review by borja-munoz — APPROVED** [review]
Ready to merge after minor changes

File: deck.gl/examples/pure-js/vue/src/components/map-component/map-utils/layerManager.js [hunk 1]
```diff
@@ -18,39 +18,42 @@ export default {
...
   isVisible (id) {
     // visible = undefined is treated by deckgl as visible = true
-    return this.layers[id] && this.layers[id].visible !== false
+    return this.layers[id] && this.layers[id].props.visible !== false
   },
   updateDeckInstance () {
     if (!this.deckInstance) {
       return
     }
+    const layers =[ ...Object.values(this.layers) ];
```
Comment (AdriSolid) [inline deck.gl/examples/pure-js/vue/src/components/map-component/map-utils/layerManager.js]: Can we remove the spaces? Like:
```javascript
const layers = [...Object.values(this.layers)];
```

File: deck.gl/examples/pure-js/vue/src/components/map-component/map-utils/layerManager.js [hunk 2]
```diff
@@ -18,39 +18,42 @@ export default {
...
     this.layers[layer.id] = layer
     this.updateDeckInstance()
   },
-  updateLayer (id, data) {
+  updateLayer (id, props) {
     if (!this.layers[id]) {
       return
     }
+    const layer = this.layers[id];
+    this.layers[id] = layer.clone(props);
```
Comment (AdriSolid) [inline deck.gl/examples/pure-js/vue/src/components/map-component/map-utils/layerManager.js]: Remove semicolons?

File: deck.gl/examples/pure-js/vue/src/components/map-component/map-utils/layerManager.js [hunk 3]
```diff
@@ -18,39 +18,42 @@ export default {
...
+    /*
     this.layers[id] = {
       ...this.layers[id],
       ...data
-    }
+    }*/
```
Comment (AdriSolid) [inline deck.gl/examples/pure-js/vue/src/components/map-component/map-utils/layerManager.js]: I would remove commented code

File: deck.gl/examples/pure-js/vue/src/components/map-component/map-utils/layerManager.js (hunk 1 above)
Comment (borja-munoz) [reply deck.gl/examples/pure-js/vue/src/components/map-component/map-utils/layerManager.js]: Spaces removed

File: deck.gl/examples/pure-js/vue/src/components/map-component/map-utils/layerManager.js (hunk 2 above)
Comment (borja-munoz) [reply deck.gl/examples/pure-js/vue/src/components/map-component/map-utils/layerManager.js]: Semicolons removed

File: deck.gl/examples/pure-js/vue/src/components/map-component/map-utils/layerManager.js (hunk 3 above)
Comment (borja-munoz) [reply deck.gl/examples/pure-js/vue/src/components/map-component/map-utils/layerManager.js]: Comments removed

**Pull request merged**